packages/worker/src/worker-options.ts

```diff
@@ -52,6 +52,12 @@ export interface WorkerOptions {
    */
   workflowBundle?: WorkflowBundle;
 
+  /**
+   * @ignore
+   * Advanced option that allows setting the typescript loader context path for webpack.
+   */
+  typescriptContextPath?: string;
```

Comment:
Not that I was able to find. The TS loader is useful for local development of new features, since it allows you to avoid needing separate build/test paths. Having this hidden in the docs seems sufficient.

Reply:
In the sdk-features PR I mentioned that you can pre-compile typescript, let's do that instead of adding the extra options.